OrbitGl/Track.cpp

```diff
@@ -170,6 +175,15 @@ void Track::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset) {
 void Track::UpdatePrimitives(uint64_t /*t_min*/, uint64_t /*t_max*/, PickingMode /*  picking_mode*/,
                              float /*z_offset*/) {}
 
+void Track::SetPinned(bool value) {
+  pinned_ = value && absl::GetFlag(FLAGS_thread_ordering_feature);
+  if (pinned_) {
+    picking_enabled_ = false;
+  } else {
+    picking_enabled_ = absl::GetFlag(FLAGS_thread_ordering_feature);
+  }
+}
+
```

Comment:
The value of GetFlag could be stored in a local variable. Or maybe a different flag was meant to be checked here? I find it a bit difficult to grasp the logic.

Reply:
You're right, changed this to be more clear, thanks